Win10.ps1

```diff
@@ -102,6 +104,9 @@ $tweaks = @(
 	### Explorer UI Tweaks ###
 	"ShowKnownExtensions",          # "HideKnownExtensions",
 	"ShowHiddenFiles",              # "HideHiddenFiles",
+	# "EnableFolderSeparateProcess" # "DisableFolderSeparateProcess",

```

Comment:
A comma is missing between the function names on this and next line.  
Also the name is too long. Ty to give names have up to 25 chars. In rare cases, I'll allow up to 27, but yours has 28. Shorten it, please. `EnableFldrSeparateProcess`, `EnableFolderSeparateProc`, `EnableFolderSepProcess`, a combination of thereof or anything like that to keep that name meaningful (at least for people who have a rough idea that such option exists in the GUI)